### PR TITLE
HOLD UNTIL June 15 - Update whats-new-in-defender-for-office-365.md - Add 'Configurable impersonation protection and apply Preset to entire org'

### PR DESCRIPTION
**Pull request by urgandhi**
**UPDATE: Feature is released WW 100% as of 6/10 - PR can be merged.**
Merge this PR only after I confirm the feature is released WW 100% - tentative ETA by 6/15

### CODE REVIEW

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Defender for Identity](/defender-for-identity/whats-new)
 - [What's new in Microsoft Cloud App Security](/cloud-app-security/release-notes)
 
+## June 2022
+
+- [Configurable impersonation protection custom users and domains and increased scope within Preset policies](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/configurable-impersonation-protection-and-scope-for-preset/ba-p/3294459):
+•	(Choose to) Apply Preset Strict/Standard policies to entire organization and avoid the hassle of selecting specific recipient users, groups, or domains, thereby securing all recipient users of your organization. 
+•	Configure impersonation protection settings for custom users and custom domains within Preset Strict/Standard policies and automatically protect your targeted users and targeted domain against impersonation attacks.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md:49]:
```suggestion
- [Configurable impersonation protection custom users and domains and increased scope within Preset Security policies](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/configurable-impersonation-protection-and-scope-for-preset/ba-p/3294459):

•	(Choose to) Apply Preset Strict/Standard policies to the entire organization and avoid the hassle of selecting specific recipient users, groups, or domains, thereby securing all recipient users of your organization. 

•	Configure impersonation protection settings for custom users and custom domains within Preset Strict/Standard policies and automatically protect your targeted users and targeted domain against impersonation attacks.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @urgandhi, kindly commit the suggested changes. Many thanks.

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md (hunk 1 above)
Comment (MSFTTracyP) [reply microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md:49]: Confirmed with Urja. We are good to go on 6/16.

**Review by MSFTTracyP — APPROVED** [review]
Publish

**Review by MSFTTracyP — APPROVED** [review]
Approved.

**Comment by traya1** [comment]
close/re-open PR

**Pull request merged**